src/twisted/newsfragments/9930.removal

```diff
@@ -0,0 +1 @@
+``defer.returnValue`` has been deprecated.
```

Comment:
The convention ask for the full method location.

Maybe also hint for the replacement.

```suggestion
twisted.internet.defer.returnValue has been deprecated. You can replace it with the standard `return` statement.
```

Reply:
Thanks.